### ISSUE TITLE
Nicky on the practice of keeping a Beeminder journal

### ISSUE THREAD

**Issue by dreeves** [beeminder/blog#497]
```[tasklist]
### Desiderata
- [ ] Have someone read a draft
- [ ] Keywords/tags
- [ ] Title image
- [ ] Link preview excerpt
- [ ] Publish
- [ ] Blog blurbs
- [ ] Waggledance
- [ ] Tweet
- [ ] Tip of the day
```

NICKY: For next time there's a blog emergency, I do actually have a draft written for a blog post on the practice of keeping a Beeminder journal!

_**Verbata**: nicky, journaling, beeminder case studies, getting the most out of beeminder,_

**Comment by shanaqui** [comment]
Formatted for ease-of-reading here, so I could preview it and read it over. Text file with the current master text attached; it has line breaks after most punctuation, to help me spot run-on sentences, so it's not very readable as-is.

[Beeminder Journal.txt](https://github.com/user-attachments/files/15889124/Beeminder.Journal.txt)

Things I should probably do:

* add a link to/graph image of the beeminder goal that tracks doing this: https://www.beeminder.com/shanaqui/weeklyreview
* add some kind of text about beeminding it
* think of some other images or pick pullquotes

---

I've kept a Beeminder journal in the forum for some years now. I don't remember who initially inspired me to do it, but it's a practice I've faithfully kept up every week since February 2019. You can check out all my Beeminder journals still -- they've been separated into separate threads by year of late, to keep them fresh, but you can find all of them in the [first post of the most recent thread](https://forum.beeminder.com/t/shanaquis-beeminder-journal-2024/11349). 

## But what is a Beeminder journal?

Good question! And the annoying slippery answers are "it's what you want it to be" and "whatever you feel you need". (Those might not be quite the same thing -- we don't always want what's good for us, after all!) 

The longer answer is that usually it's used to keep track of your Beeminder goals in some way. Perhaps it's a weekly or monthly record of your derailments, or a progress log of the things you've achieved. Maybe it's the process that works for you (in which case, a private record works) or maybe you need the accountab­ility on the forum, where other Beeminder users can see (and comment). 

## How I use my Beeminder journal

Over time, I've done a few different things with my Beeminder journal. The constants have been that I post on a weekly basis, and I write about my derailments, the new goals I've created (if any), and the changes I've made to my goals. 

In fact, that's made it tie in beautifully with another concept: calendialling. We have [a whole blog post about it](https://blog.beeminder.com/calendial/), but in short, calendialling is all about settling down with your calendar and your Beeminder goals and setting the breaks or rate changes you need to fit in with upcoming events. That might mean dialling up your study goal before an exam, for example, or reducing the rate to gain buffer to cover a vacation. It might mean outright putting in breaks with rates of 0/day. 

So each week, writing in my Beeminder journal has been a ready-made reminder to set the breaks I need. It's also a good way to prompt regular reflection on your goals. Are the rates right? Is the pledge high enough? Is this goal helping me? 

Of late, I've been setting myself an intention for the week ahead at the end of each review. Thanks to that, I've reviewed all my pledge caps, reviewed all my goal rates, planned out new goals, reviewed whether my current goals are necessary, and staked out space for catching up with my goals after realising I was a bit snowed under. 

Aaand I've remembered to put breaks in on time, too. Mostly.

## What I would recommend

Over the years of doing it,  I've obviously come up with some preferences and ideas about how a Beeminder journal works best. My first rule though is *flexibility*. If you look at how my Beeminder journals have been set up through the years, they've taken different forms. At first I didn't use headings, and *which* headings I've used has varied a lot over time. If something doesn't work for you, ditch it! 

That said, how do you figure out what might work for you when you're starting out? I'd sit down and figure out what you think you want to record: maybe you want to keep track of derailments, so you'd have a derailments section the same way I do; maybe you want a prompt to check in on your calendialling every week, in which case maybe you actually have a section just for that. If you want to set a goal for yourself each week, like "this week I will: review my goal rates", then make a section for that! If there's something that prompts you to create a lot of transient goals (like I have in the past with my reading goals) then a section for that might help keep things tidier, and easier to refer back to in future.

Another thing to think about is how often you want to do this review. For some, once a week is way too frequent -- with only a handful of goals, sometimes you just don't have anything to talk about after a week. I probably wouldn't recommend more often than once a week to *ensure* you check in, but maybe for you it's worth using the space to talk about any resistance you're encountering in doing your goals, or any relevations that you strike you during the week, so you want to do it a bit more often than once a week. As with the content of your journal, in the end it's all about what you might find useful.

And finally, you probably need to think about whether you're doing this publicly or not. I do, because I don't have much I keep under my hat, and because I find the accountability helpful. You might prefer to do it on paper, or in a private journal somewhere.

Again, it's all down to what you find useful, *and* what you find comfortable. You're the person who has to use it, after all!

## To sum up:

A Beeminder Journal makes a good complement to thoughtful beeminding. It's a good way to notice trends, notice stuff that you're struggling with, and also keep you touching base with your goals regularly. Sometimes when things are going smoothly, there isn't a lot to say. Sometimes when times are rocky, you'll need to write more. 

In the end, your Beeminder journal might not look like mine (or anyone else's) and that's fine! But it's definitely a practice I personally recommend.

**Comment by bsoule** [comment]
![bee ness_an_etching_of_a_monk_diligently_illuminating_a_bee_jou_cb88df73-ae25-4be5-b3d2-270c961108aa](https://github.com/beeminder/blog/assets/46881/b6d2a1d9-fc99-4493-83b8-53366c670054)

**Comment by bsoule** [comment]
![bee ness_an_etching_of_a_monk_painting_an_illuminated_a_bee_man_ed36976c-bcf5-4904-aa57-a6e5f417a505](https://github.com/beeminder/blog/assets/46881/9ec6c19b-6183-4a7f-b6dc-31527a19937d)

**Comment by dreeves** [comment]
![image](https://github.com/beeminder/blog/assets/9928/84e3f22d-c17c-4fda-a48c-e0d2ec852ef2)
